exercises/practice/ordinal-number/OrdinalNumber.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+declare(strict_types=1);
+
```

Comment:
Let's sort this out in the issue before including it in this PR

```suggestion
declare(strict_types=1);

```

Reply:
Removed till clarified.